src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -12,6 +12,7 @@
 using System.Collections.Concurrent;
 using Microsoft.Data.SqlClient;
 using System.Threading.Tasks;
+using Microsoft.SqlTools.ServiceLayer.SchemaCompare.Contracts;
```

Comment:
While you're here, can you sort the usings properly?  There's a VS setting to do it for you automatically (system first, then the rest alphabetically)

Reply:
Same for other files, please!